system-design/margo-api-reference/workload-api/onboarding-api/device-onboarding.md

```diff
@@ -0,0 +1,16 @@
+# Device Onboarding
+
+> Action: This API needs to be defined
```

Comment:
Same general comment as raised on teh rootca-download artifact.
While its not part of the PR, I would have a question around whether we should consider a dedicated Onboarding service? The WOS appears to have that responsiblity so far. But I think onboarding a device may have different actions based on policy. For example, onboarding may trigger a flashing of a new distro. Perhaps an audit and burn in test, or other actions we may not think of today. Such a service might have a plugin mechanism that talks east-west to WOS and other services to coordinate action.

Reply:
We are currently discussing the Device Orchestration onboarding within the Discord post. See post titled Margo device management interface to continue the discussion.